richmemo.pas

```diff
@@ -268,6 +282,7 @@   TCustomRichMemo = class(TCustomMemo)
     property OnPrintAction: TPrintActionEvent read fOnPrintAction write fOnPrintAction;
     property OnLinkAction: TLinkActionEvent read fOnLinkAction write fOnLinkAction;
     property CanRedo: Boolean read GetCanRedo;
+    property FastLoad: boolean read fFastLoad write fFastLoad;
```

Comment:
"FastLoad" property is my biggest concern. Doesn't look like a nice API design.
I can understand if it's added as a parameter to LoadRTF, but adding it as a property seems awkward to me,
Because It doesn't affect the behavior of the control in general, it only comes into effect when loading data.

Reply:
Indeed, FastLoad was not supposed to be exposed to the user, I will remove it and replace it with a developer flag, I'm not ready to remove the previous default code because I intended to do some tests and want it for reference. The chunked (fast) loading is what will remain.